projects/github-actions/repo-gardening/src/utils/labels/get-all-labels.js

```diff
@@ -0,0 +1,43 @@
+/* global GitHub */
+const debug = require( '../debug' );
+
+// Cache for getLabels.
+const cache = {};
+
+/**
+ * Get all the labels available in the repo.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} filter  - Optionally filter to only return a subset of labels. Use a regex pattern.
```

Comment:
This seems more like what you intend.
```suggestion
 * @param {RegExp|string} filter  - Optionally filter to only return a subset of labels. Use a regex pattern.
```